### ISSUE TITLE
Mozzie.param does not have a server

### ISSUE THREAD

**Issue by Althouse-D** [sarsky/Mozzie#2]
Mozzie.param does not connect to a server. It will not download. 

What parameters need to be changed on mission planner?

**Comment by aj265** [comment]
Indeed. Did you receive the mozzie.params meanwhile or figured out your own ones?
